src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepository.java

```diff
@@ -25,6 +25,8 @@
 
     List<Envelope> findByStatus(Status status);
 
+    List<Envelope> findByZipFileName(String zipFileName);
```

Comment:
I guess Exela generates different file name for two different zip files uploaded to two different containers at same time otherwise it should be by zip file name and container/jurisdiction ?

Reply:
the story mentions retrieving status by file name only, not jurisdiction, hence the list.